lib/view_component/base.rb

```diff
@@ -241,6 +241,9 @@ def content_evaluated?
     #
     mattr_accessor :render_monkey_patch_enabled, instance_writer: false, default: true
 
+    # Configure if previews should include example source or not.
+    mattr_accessor :show_previews_source, instance_writer: false, default: false
```

Comment:
Now that we have yard documentation (see #968) I guess we should add a yard-style documentation block here as well.

Can you confirm @joelhawksley?

Reply:
Yes, please do ☝🏻, and be sure to run `rake docs:build` after you do 😄

Reply:
:+1: 

@johannesengl let me know if you want me to wrap this up for you!

Reply:
@Spone Sure feel free to wrap this up. Thanks!